modules/gitops-release-notes-1-4-13.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+
+[id="gitops-release-notes-1-4-13_{context}"]
+= Release notes for {gitops-title} 1.4.13
+
+{gitops-title} 1.4.13 is now available on {product-title} 4.8, 4.9, 4.10 and 4.11.
+
+[id="fixed-issues-1-4-13_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* Before this update, in a large set of applications the application controllers were restarted multiple times due to the unresponsiveness of liveness probes. This update fixes the issue by removing the liveness probe in the application controller `StatefulSet` object. link:https://issues.redhat.com/browse/GITOPS-2354[GITOPS-2354]
```

Comment:
The description for [GITOPS-2354](https://issues.redhat.com/browse/GITOPS-2354) is wrong, this was fixed in 1.4.12- [GITOPS-2153](https://issues.redhat.com/browse/GITOPS-2153)
I think the description for [GITOPS-2354](https://issues.redhat.com/browse/GITOPS-2354) should be similar to the description of [GITOPS-2353](https://issues.redhat.com/browse/GITOPS-2353) and [GITOPS-2352](https://issues.redhat.com/browse/GITOPS-2352)